app/src/main/java/com/ku_stacks/ku_ring/navigator/KuringNavigatorImpl.kt

```diff
@@ -22,15 +22,8 @@ import com.ku_stacks.ku_ring.ui_util.KuringNavigator
 import javax.inject.Inject
 
 class KuringNavigatorImpl @Inject constructor() : KuringNavigator {
```

Comment:
이거 ActivityComponent에서 선언하고 아래와 같이 바꾸는건 어떨까요...?

```suggestion
class KuringNavigatorImpl @Inject constructor(
    private val activity: Activity
) : KuringNavigator {
```

Reply:
Activity 외에도 `MyFirebaseMessagingService`나 `ReEngagementNotificationWork`에서 Intent를 얻기 위해 navigator를 사용하고 있긴 합니다. 

Intent 생성 코드를 다른 클래스로 옮겨도 되지만, 향후 single activity를 도입하면 Intent를 만들 필요가 없어지므로 activity나 context를 매개변수로 받을 이유도 없어집니다.

그래서 제 생각은 `굳이?` 입니다.

Reply:
아하 오케이! 이건 넘어가시죠